components/collection/CollectionInfo.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div
+    class="is-flex is-justify-content-space-between mobile-flex-direction-column gap">
+    <div class="is-flex is-flex-direction-column is-flex-grow-1 max-width">
+      <div v-if="collectionInfo?.currentOwner" class="is-flex mb-2">
+        <div class="mr-2">{{ $t('activity.creator') }}</div>
+        <nuxt-link :to="`/${urlPrefix}/u/${address}`">
+          <IdentityIndex
+            ref="identity"
+            :address="address"
+            link-class="has-text-link" />
+        </nuxt-link>
```

Comment:
```suggestion
        <nuxt-link :to="`/${urlPrefix}/u/${address}`" class="has-text-link">
          <IdentityIndex
            ref="identity"
            :address="address" />
        </nuxt-link>
```

Reply:
I'm afraid this alone is not enough
on mobile, and maybe other cases, i did not explore deeply, ```IdenitityIndex``` renders ```IdentityLink``` and the applied class on ```nuxt-link``` just doesn't work

I've solved it before using css in ```collectionInfo``` which you requested changed

so this time i pass a prop through

on desktop, or any other case where ```IdentityIndex``` renders ```IdenitityPopover``` your solution is required

Reply:
is it still a problem?